storage/local/test_helpers.go

```diff
@@ -65,3 +65,7 @@ func NewTestStorage(t testutil.T, encoding chunkEncoding) (*MemorySeriesStorage,
 
 	return storage, closer
 }
+
+func mkFpSeriesPair(s *MemorySeriesStorage, fp model.Fingerprint) fingerprintSeriesPair {
```

Comment:
Capitalization nit: `mkFPSeriesPair`. But given the name of the returned type (where FP is not abbreviated), and given that "make" is rarle abrreviated "mk" in typical Go constructors, why not call it `makeFingerprintSeriesPair` after all?